### ISSUE TITLE
[Feature] Icons doesn't scale with text enlargement

### ISSUE THREAD

**Issue by hildeor** [AtB-AS/mittatb-app#509]
## Origin
[Usability test with Blindforbundet, 23.09.2020, finding 19.
](https://github.com/AtB-AS/innsikt/blob/master/brukertester/2020-09-23_Blindeforbundet.md#19)

> Ikoner skalerer ikke med tekstforstørring.

## Motivation
_A short description of what user needs or business goals this feature will solve._

## Hypotheses and assumptions
_A list of hypotheses and assumptions we have made about the user or the proposed solution._

## Proposed solution
Define how icons should behave with text enlargement, including spacing between objects.

Issue also related to #495

**Comment by hildeor** [comment]
Håndteres via #1047
